src/main/kotlin/racingcar/validator/Validator.kt

```diff
@@ -0,0 +1,25 @@
+package racingcar.validator
+
+import racingcar.Messages
+
+object Validator {
+    private const val MAX_NAME_LENGTH = 5
+
+    fun validateTries(tries: String): Int {
+        require(tries.toIntOrNull() != null) { Messages.NOT_INTEGER }
+
+        val triesInt = tries.toInt()
+        require(triesInt > 0) { Messages.NOT_SMALLER_THAN_ZERO }
+
+        return triesInt
+    }
+
+    fun validateCarNames(carNames: List<String>) {
+        carNames.forEach {
+            require(it.isNotEmpty()) { Messages.NOT_EMPTY_NAME }
+        }
+        carNames.forEach {
+            require(it.length <= MAX_NAME_LENGTH) { Messages.NOT_BIGGER_THAN_FIVE }
+        }
```

Comment:
forEach를 두 번 사용하셨는데 한 번으로 합치면 조금 더 효율적이고 읽기 쉬울 것 같습니다!
```suggestion
        carNames.forEach {
            require(it.isNotEmpty()) { Messages.NOT_EMPTY_NAME }
            require(it.length <= MAX_NAME_LENGTH) { Messages.NOT_BIGGER_THAN_FIVE }
        }
```

Reply:
아 그렇네요! 조언 감사합니다